07_wait_for_cluster.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+source logging.sh
+source common.sh
+
+export KUBECONFIG=ocp/$CLUSTER_NAME/auth/kubeconfig
+
+workers=$NUM_WORKERS
+if $APPLY_EXTRA_WORKERS ; then
+    workers=($workers + $NUM_EXTRA_WORKERS)
+fi
+if (( $workers <= 0 )); then
+    # we need some workers to have all the operators up.
+    exit 0
+fi
+
+while [ "$(oc get clusteroperators -o json | jq '.items[].status.conditions[] | select(.type=="Available") | select(.status!="True")')" != "" ]; do
```

Comment:
Might be cleaner to do something like `oc wait --for=condition=Available=True  clusteroperators --all` here?